networkit/graph.pyx

```diff
@@ -571,6 +571,11 @@ cdef class Graph:
 	def iterEdges(self):
 		"""
 		Iterates over the edges of the graph.
+		
+		The iteration is done from node to node, iterating over all edges
+		of a node before passing to edges of the next node in the order of node Ids.
```

Comment:
Maybe rephrase with "For each node u in the graph in ascending node id order, the iterator yields the edges incident to u".